wandb/apis/public/api.py

```diff
@@ -369,49 +369,28 @@
         base_url: str,
         init_api_key: str | None = None,
     ) -> str:
-        """Attempts to load a configured API key or prompt if one is not found.
-
-        The API key is loaded in the following order:
-            1. User explicitly provided api key
-            2. Thread local api key
-            3. The session api key
-            4. Environment variable
-            5. Netrc file
-            6. Interactive prompt (if possible)
```

Comment:
We should document this somewhere... Where did you want to put it?

Reply:
It's the behavior of `login()`. This random private function in Api is not the right place to document it :p

I can update the `_login()` docs for now.

Reply:
yea, I agree - please do.

Reply:
Updated the doc for `wandb.login()`. Take a look at that "beauty"